### DISCUSSION TITLE
My codespace suddenly doesnt working

### DISCUSSION THREAD

**Discussion by Ainol1995** [community/community#38266] [Codespaces]
![2BC4C52A-73A2-4B4C-BFE2-C961842B3ED2](https://user-images.githubusercontent.com/117441062/199944842-d718cb49-76e2-422b-94c3-4c37c599c723.jpeg)
I cant access to my terminal. Im using remote codespace by cs50. I already delete the codespace also github account. When i change theme, the terminal section become white. Does anyone know about this? I currently taking course is cs50 harvard.

**Comment by jfullstackdev** [comment]
I'm seeing a lot of cs50 bugs being asked here in the Community 

maybe you can ask at their Org page too

https://github.com/cs50

**Comment by fara-nak** [comment]
hi @Ainol1995  seems like a vscode issue as reported here: https://github.com/microsoft/vscode/issues/165416
you can also comment there and let them know

**Reply by fara-nak** [reply]
also from your screenshot i couldn't say if its just your terminal that is not working or your codespace fully stopped working.
